smallLibraries/Google.Cloud.Templates/content/gcpmvc/CSharp/Controllers/HomeController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+
+namespace MyGcpMvcProject.Controllers
+{
+    public class HomeController : Controller
```

Comment:
Is it worth having a comment here to explain that this is the default controller? Or do you think that's too obvious?